### PR TITLE
Converting scripts

### PR DESCRIPTION
**Pull request by Talamar1**

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizen/events/player/PlayerConsumesScriptEvent.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import org.bukkit.event.player.PlayerItemConsumeEvent;
+
+import java.util.HashMap;
+
+public class PlayerConsumesScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // player consumes item
+    // player consumes <item>
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerConsumesScriptEvent.java]: Could totally add `in <area>` to this event. And like... almost every event really.

Comment (Talamar1) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerConsumesScriptEvent.java]: Agreed, looking to do in:area for all new events as soon as you code it :)
Also, we need a meta section to describe what "area" can mean (world, cuboid, etc...)

Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerConsumesScriptEvent.java]: Just use`in <area>` with the existing engine

File: src/main/java/net/aufdemrand/denizen/events/block/BlockSpreadsScriptEvent.java [hunk 2]
```diff
@@ -28,7 +28,7 @@
     //
     // @Context
     // <context.location> returns the dLocation the block.
-    // <context.material> returns the dMaterial of the block.
+    // <context.material> DEPRECATED
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/block/BlockSpreadsScriptEvent.java]: STOP DEPRECATED CONTEXT.MATERIAL EVERYWHERE `D:<`

File: src/main/java/net/aufdemrand/denizen/events/player/PlayerPlacesHangingScriptEvent.java [hunk 3]
```diff
@@ -30,7 +31,7 @@
     //
     // @Context
     // <context.hanging> returns the dEntity of the hanging.
-    // <context.location> returns the dLocation of the block the hanging was placed on.
+    // <context.location> DEPRECATED
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerPlacesHangingScriptEvent.java]: Should not be deprecated.

Comment (Talamar1) [inline src/main/java/net/aufdemrand/denizen/events/player/PlayerPlacesHangingScriptEvent.java]: I already fixed them all...

**Pull request merged**